tests/unit/test_cursorManager.py

```diff
@@ -47,6 +49,32 @@ def test_unselectPrevChar(self):
 		cm.script_selectCharacter_back(None) # "a" unselected
 		self.assertEqual(cm.selectionOffsets, (0, 0)) # Caret at "a", no selection
 
+	def test_unselectNextChar(self):

```

Comment:
I don't really understand the names "unselectPrevChar" and "unselectNextChar" somehow they seem the wrong way around? They might be clearer if called "selectPreviousUnsel" and "selectNextUnsel"